client-dynamodb-v2/src/main/kotlin/app/cash/backfila/client/dynamodbv2/DynamoDbBackfill.kt

```diff
@@ -87,4 +88,9 @@ abstract class DynamoDbBackfill<I : Any, P : Any> : Backfill {
 
   /** See [ScanRequest.setExpressionAttributeNames]. */
   open fun expressionAttributeNames(config: BackfillConfig<P>): Map<String, String>? = null
+
+  /**
+   * Override this to do any work after the backfill completes.
+   */
+  open fun finalize(config: FinalizeBackfillConfig<P>) {}
```

Comment:
finalize() is a deprecated method on Java's root Object class that, historically, was invoked prior to garbage collection. I don't know if Kotlin has any impact here, but I don't think that's a function name we should be using. I'd also avoid any use of the term finalize. Maybe onComplete?

Reply:
I think finalize is fine with the Parameter as well.